modules/cpmso-config-options.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cpmso-config-prefix_{context}"]
+= Adding a custom prefix to control plane machine names
+
+You can customize the prefix of machine names created by a control plane machine set.
+This can be done by editing a control plane machine set custom resource (CR)
+
+.Procedure
+
+// Saw the step below in another procedure, let me know if it's good to keep it in here.
+
+. Edit your control plane machine set CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit controlplanemachineset.machine.openshift.io cluster \
+  -n openshift-machine-api
+----
+
+. Add the following snippet to the `ControlPlaneMachineSet` CR:
++
+[source,yaml]
+----
+spec:
+  machineNamePrefix: <machine_prefix>
+----
++
+where `<machine_prefix>` specifies a prefix name that satisfies the following requirements:
+
+** The name contains no more than 253 characters
+** The name contains only lowercase alphanumeric characters, `-`, or `.`
+** The name starts and ends with an alphanumeric character
+
+. Save your changes to the `ControlPlaneMachineSet` CR.
+
+. If you are using an `OnDelete` update strategy for your control plane machine set: Manually apply the prefix to the name of any existing machines.
```

Comment:
If there are verification steps to add, please let me know. 

For example, what command to run to check machine names, and example output of that command